debian/control

```diff
@@ -0,0 +1,20 @@
+Source: dracut-sshd
+Section: contrib/utils
+Priority: optional
+Maintainer: Georg Sauthoff <mail@georg.so>
```

Comment:
Is this the maintainer of the software itself or the debian package?

Reply:
It indeed should be the package maintainer's information. I have changed it to my info. Please confirm whether this works for you.

Reply:
Yes, I don't have any plans for maintaining a debian package.